docs/content/reference/video.md

```diff
@@ -24,25 +24,30 @@ This gives the best compression ratio, reducing file sizes and bandwidth require
 snippet: archetypes/video_auto_frames
 
 ## `AssetVideo` limitations
-Video support is new in Rerun, and has several limitations:
+Video support is new in Rerun, and has a few limitations:
 
-* [#7298](https://github.com/rerun-io/rerun/issues/7298): Video playback only works in the web viewer
 * [#7354](https://github.com/rerun-io/rerun/issues/7354): Only the MP4 container format is supported
+* [#7298](https://github.com/rerun-io/rerun/issues/7298): Only the AV1 codec is supported in the native viewer
 * [#5181](https://github.com/rerun-io/rerun/issues/5181): There is no audio support
+* [#7594](https://github.com/rerun-io/rerun/issues/7594): HDR video is not supported
 * There is no video encoder in the Rerun SDK, so you need to create the video file yourself
-* Only a limited sets of codecs are supported (see below)
+* A limited sets of codecs are supported on web (see below)
 
-## Web viewer support
-As of writing, playback of `AssetVideo` is only supported on the web viewer.
-Native video playback is coming, and can be tracked [in this GitHub issue](https://github.com/rerun-io/rerun/issues/7298).
+## Streaming video
+Rerun does not yet support streaming video support. You can work around this limitation by logging many small `AssetVideo`s to the same Entity Path. See [#7484](https://github.com/rerun-io/rerun/issues/7484) for more.
```

Comment:
```suggestion
Rerun does not yet support streaming video support. For scenarios where you don't need live video, you can work around this limitation by logging many small `AssetVideo`s to the same Entity Path. See [#7484](https://github.com/rerun-io/rerun/issues/7484) for more.
```

I think we shouldn't pretend it's a replacement if you want live video streaming in. That's completely unfeasible with that approach and I'd rather don't have people try